### PR TITLE
fixed snappy loading and updated docs

### PR DESCRIPTION
**Pull request by SebastianDremo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Instructions for other distributions such as CentOS, Fedora or Debian are availa
 *	Mac https://www.microsoft.com/net/download?initial-os=macos
 * `brew install gmp && brew install snappy && brew install lz4 && brew install zstd`
 * if you have issues on launch you may need to call
-`brew install rocksdb`
+`brew install rocksdb` or add `<UseAppHost>true</UseAppHost>` to `<PropertyGroup></PropertyGroup>` in Nethermind.Runner.csproj
```
Comment (tkstanczak) [inline README.md:57]: not specific enough - where people can find it, at what step they should do it
does not explain what each option does and what are the consequences of AppHost

File: docs/source/build.rst [hunk 2]
```diff
@@ -61,6 +61,13 @@ MacOS
 ::
 
     brew install gmp && brew install snappy && brew install lz4
+
+in some cases in order to load snappy correctly you might have to add in Nethermind.Runner.csproj
```
Comment (tkstanczak) [inline docs/source/build.rst:65]: where is Nethermind.Runner.csproj

File: src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj [hunk 3]
```diff
@@ -7,6 +7,7 @@
     <ConcurrentGarbageCollection>true</ConcurrentGarbageCollection>
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
     <Nullable>enable</Nullable>
+    <UseAppHost>true</UseAppHost>
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj:10]: do you want to always add it?
what are the consequences?
(I assume it was committed by mistake)

Comment (SebastianDremo) [reply src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj:10]: yeah, forgot to unstage it

**Pull request closed without merging**